### PR TITLE
Feature/kobaj add note api

### PR DESCRIPTION
**Pull request by kobaj**
This is how we intend on marking a registration as like "checked in" right? IE, the message would be something like "Badge Printed!" maybe with some metadata about which station or printer did the operation. Do I understand it correctly?

### CODE REVIEW

**Comment by Cellivar** [comment]
> This is how we intend on marking a registration as like "checked in" right?

There's actually a separate thing for that: 

image

And a (real! public!) API for doing so: https://docs.webconnex.io/api/v2/#check-in-by-id

This will be useful for marking reprints though!

File: src/regfox/payloads/regfox_api_add_note.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+/**
+ * @return {*} a body object that can add a note to a registration.
+ *
+ * @param {string} message that should be seen as a note
+ * @param {string} id of registration (**not** the RegistrationId)
+ */
+const buildAddNoteBody = (message, id) => {
+  // I have no idea what these values mean and can't find any documentation on them.
+  const noteType = 5;
+  const createdBy = 144247; // Seems we can set this to any value?
```
Comment (Cellivar) [inline src/regfox/payloads/regfox_api_add_note.js:11]: Maybe the UserID of the person creating the note?

**Review by Cellivar — APPROVED** [review]
Thanks for adding this!

File: src/regfox/payloads/regfox_api_add_note.js (hunk 1 above)
Comment (kobaj) [reply src/regfox/payloads/regfox_api_add_note.js:11]: Not sure how to get that at the moment. But I'll make an action item to look into it!

**Pull request merged**